app/controllers/home_controller.rb

```diff
@@ -1,4 +1,12 @@
+# frozen_string_literal: true
+
 class HomeController < ApplicationController
   def index
+    if current_user.present?
+      unless current_user.status
+        sign_out
+        redirect_to sign_in_path
+      end
+    end
```

Comment:
I think we can get rid of this logic just by using the `before_action` `require_login`. 
More info about this `action` here: [Clearance Access Control](https://github.com/thoughtbot/clearance#access-control)

But just curious.
Why is this necessary?
I thought we were looking for an interface that supports both, logged and unlogged users. But I can be wrong.

Reply:
Oh sorry, I forgot that but it is ready 😅 

This is necessary because with this, a "deleted" user should not login

Reply:
I'm going to leave it as it was, beacuase our teammate noticed that the `before_action require_login` doesn't allow to see the main page unless the user is logged in, sorry for not having seen that detail 😅